### PR TITLE
Adding basic logging

### PR DESCRIPTION
**Pull request by wtgee**
I've thrown in a bunch of logging statements as I need to them to help debug some hardware. This will want to be improved some.

Related to #21

### CODE REVIEW

File: domehunter/dome_control.py [hunk 1]
```diff
@@ -466,8 +498,12 @@ def _az_to_ticks(self, az):
             Returns encoder tick count corresponding to dome azimuth.
 
         """
+        logger.debug(f'Home azimuth: {self.home_az} Convert Azimuth: {az}')
         az_relative_to_home = (az - self.home_az).wrap_at(360 * u.degree)
-        return az_relative_to_home.degree / self.degrees_per_tick.degree
+        logger.debug(f'Azimuth relative to home: {az_relative_to_home}')
+        encoder_ticks = az_relative_to_home.degree / self.degrees_per_tick.degree
+        logger.debug(f'Encoder ticks for requested az: {encoder_ticks}')
+        return encoder_ticks
```
Comment (AnthonyHorton) [inline domehunter/dome_control.py:491]: Should this be forced to be an int?

**Review by AnthonyHorton — APPROVED** [review]
LGTM

File: domehunter/dome_control.py (hunk 1 above)
Comment (wtgee) [reply domehunter/dome_control.py:491]: I thought I had recommended that in the original PR and there was some reason not to. Although maybe that was something else.

Regardless, I was trying to change as little as possible in this PR other than adding logging. I split it up into a bunch of lines just to log along the way.

But yes, I would think it should be.

**Comment by wtgee** [comment]
@fergusL did you want to take a look at this or should I go ahead and merge it and you can build on it from there?

File: domehunter/dome_control.py (hunk 1 above)
Comment (fergusL) [reply domehunter/dome_control.py:491]: Oh yea, it should have been fixed in the big pr but I must have forgot... My bad

**Comment by fergusL** [comment]
@wtgee sorry, something weird with my emails, didnt see these prs until now. Looks fine to me but do we still want to go with `logging`? Or should I change over to `logbook`? In any case can probably just merge, as I don't think it'd be hard for me to switch to using logbook for the next logging pr if thats the way we want to go

and yea, `_az_per_tick` should have been forced to return an `int` in the big pr but I just forgot to make the change

**Review by fergusL — APPROVED** [review]

**Comment by wtgee** [comment]
@fergusL I'll go ahead and merge and then you can make the switch to logbook. It will just involve changing the setup and maybe a few of the levels off of `info` to `notice`. I maybe just should have started with logbook but didn't have time to read through the docs and see what might be different.

**Pull request merged**